validator/types.bal

```diff
@@ -0,0 +1,13 @@
+type Term record {
+    string code;
+    string system;
+    string version?;
```

Comment:
let's rename `version` into `'version` (add apostrophe to the front). hence this can be used as a keyword